openapi/integrator/token-requestor/client.yaml

```diff
@@ -59,10 +59,58 @@ paths:
           $ref: '../components.yaml#/components/responses/500Error'
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
-  /payment-tokens/{tokenId}/expiry-extensions:
+  /payment-tokens/{tokenId}/resumptions:
+    description: Payment token resumption notification
+    post:
+      operationId: resumeToken
+      parameters:
+        - $ref: '../components.yaml#/components/parameters/tokenId'
+      requestBody:
+        required: true
+        content:
+          application/json;charset=UTF-8:
+            schema:
+              $ref: '#/components/schemas/TokenUpdateData'
```

Comment:
We should not send the status field anymore, but still messageId. Goes for suspension as well.